torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp

```diff
@@ -1180,15 +1316,17 @@ void InsertQuantDeQuantHelper::run(
       auto tp = getQSchemeAndQParamVector(module, n);
       checkQScheme(graph.get(), std::get<0>(tp));
       auto qparam_map = std::get<1>(tp);
-      TORCH_INTERNAL_ASSERT(
-          qparam_name_map_for_node_.count(n),
-          "Expected to have a qparam_name_map for node:",
-          *n);
-      auto qparam_name_map = qparam_name_map_for_node_.at(n);
-      for (auto& pr : qparam_map) {
-        const auto& name = pr.first;
-        const auto& qparam = pr.second;
-        module._ivalue()->setAttr(qparam_name_map.at(name), qparam);
+      if (qparam_map.size() > 0) {
```

Comment:
I think it's better to keep the insert here. breaking this insert means there is something wrong and we'll need to fix that

Reply:
when is the qparam map empty?

Reply:
Noop observer is an example of an observer that doesn't have qparam_map

Reply:
OK, it's probably fine in this case, could you also add a comment for this check?